sdk/storage/storage-blob-changefeed/test/utils/testutils.common.ts

```diff
@@ -96,7 +96,7 @@ export class SimpleTokenCredential implements TokenCredential {
 }
 
 export function isBrowser(): boolean {
-  return typeof window !== "undefined";
+  return typeof self !== "undefined";
```

Comment:
Can self also defined when it's not a browser environment?  
I searched around, seems self can exist no matter whether there's a windows context:

> By using self, you can refer to the global scope in a way that will work not only in a window context (self will resolve to window.self) but also in a worker context (self will then resolve to WorkerGlobalScope.self).

>

Reply:
`self` is for a browser environment but it can lead to the correct global object when in a web-worker or in the main process. It will not be defined in node (unlike `globalThis`) so I think that for the purpose of checking if you're in a browser environment referencing `self` should give you the expected result as far as I know.

Reply:
Got it. Thanks